### ISSUE TITLE
Adjust default of countries lived in + dates in combined questions

### ISSUE THREAD

**Issue by mnewsted** [IllinoisLegalAidOnline/docassemble-USCISApplications#202]
In combined_application_questions.yml, on id: status in other countries, I think the default here needs to be
`      ${ showifdef("users[0].lived_in_countries_traveled_through_dates") if i > 0 else '' }`
With the current code (below) it's defaulting to "True".
Here's the full field:
```
  - Which country, and what dates did you live there?: users[i].lived_in_countries_traveled_through_dates
    datatype: area
    show if: users[i].lived_in_countries_traveled_through
    default: |
      ${ showifdef("users[0].lived_in_countries_traveled_through") if i > 0 else '' }
```

**Comment by mnewsted** [comment]
Other minor tweaks:
Remove footer since not used in this scenario
Adjust I-821 - Part 2, question 5 - make checkbox style cross to be consistent with other fields

**Comment by mnewsted** [comment]
closed with PR #203
